h2o-bindings/bin/custom/python/gen_aggregator.py

```diff
@@ -0,0 +1,13 @@
+class_init_extra = """
+self._parms["_rest_version"] = 99
+"""
+
+class_extras = """
+@property
```

Comment:
Is there any chance we could define this as actual python code instead of strings?

We could have a mixin class that would be added to the generated one.

Reply:
@michalkurka yeah, I've been thinking about something like that too.
For now, my approach works for any language as these are just strings.

But for python we could do much better, especially for those extras:
I could also declare those extras as a proper mixin as you suggest (would appear as code in this file), copy the source code using `inspect` module and add the mixin in the base classes dynamically.
Will definitely try it later.

Reply:
cool, I think this is definitely a step in a good direction